app/results.py

```diff
@@ -57,12 +57,16 @@ def search(self, query):
         results = None
         if self.config.external_search_backends:
             results = external_search.search(
-                self.repo, query, MAX_RESULTS,
+                self.repo, TextQuery(query), MAX_RESULTS,
```

Comment:
Can you rename this "query" to "query_txt"? "query" in this file usually means a Query object.